execution/execution-correctness/src/local_client.rs

```diff
@@ -1,57 +0,0 @@
-// Copyright (c) The Libra Core Contributors
```

Comment:
please avoid leaking this into a massive commit, two commits here just like in safety-rules